test/Table_Tests/src/Data_Formatter_Spec.enso

```diff
@@ -137,14 +137,14 @@ spec =
         Test.specify "should format dates" <|
             formatter = Data_Formatter
             formatter.format (Date.new 2022) . should_equal "2022-01-01"
-            formatter.format (Time.new 1999 . internal_zoned_date_time) . should_equal "1999-01-01 00:00:00"
-            formatter.format (Time_Of_Day.new . internal_local_time) . should_equal "00:00:00"
+            formatter.format (Time.new 1999) . should_equal "1999-01-01 00:00:00"
```

Comment:
I'd appreciate also a test where we overwrite the zone to a different one, preferably two different ones (let's say CEST and PDT or something like these), to verify that the formatter correctly deals with zoned dates.